src/helpers/filterResultsByPrice.ts

```diff
@@ -0,0 +1,5 @@
+export default (results, price: string) => {
+  return results.filter((result) => {
+    return result.price === price;
```

Comment:
Very good. Do you know the difference between `==` and `===`? I assume you do since you've used the later, if you're not sure it's worth reading about it.